### PR TITLE
change in brand

### PR DESCRIPTION
**Pull request by jayesh-parmar**

### CODE REVIEW

File: app/Http/Controllers/BrandController.php [hunk 1]
```diff
@@ -2,46 +2,40 @@
...
 
+use App\Http\Requests\BrandRequest;
 use App\Models\Brand;
-use Illuminate\Http\Request;
 
 class BrandController extends Controller
 {
     public function index()
     {
         $brand = Brand::paginate(10);
```
Comment (rajan-parmar) [inline app/Http/Controllers/BrandController.php]: selected the required columns. changes all

File: app/Http/Controllers/ColorController.php [hunk 2]
```diff
@@ -36,19 +31,13 @@ public function edit(string $colorId)
     {
         $color = Color::find($colorId);
```
Comment (rajan-parmar) [inline app/Http/Controllers/ColorController.php]: select columns

File: app/Http/Requests/BrandRequest.php [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    public function authorize(): bool
+    {
+        return true;
+    }
+    public function rules(): array
+    {
+        $rules = [
+                'name' => ['required','max:255',Rule::unique('brands')->ignore($this->brand, 'name')],
+                ];
+
```
Comment (rajan-parmar) [inline app/Http/Requests/BrandRequest.php]: you can direct return & proper indentation

File: resources/views/admin/pages/brands/form.blade.php [hunk 4]
```diff
@@ -0,0 +1,29 @@
+@extends('admin.layouts.main')
+@section('content')
+<h4 class="mb-4 text-lg font-semibold text-gray-600 dark:text-gray-300">
+    @if (isset($brands)) Update @else Add @endif Brand
+</h4>
+@if (isset($brands))
+<form action="{{route('brand.update',$brands->id)}}" method="post">
+    @method('PUT')
+    @else
+    <form action="{{route('brand.store')}}" method="post" class="mt-5">
```
Comment (rajan-parmar) [inline resources/views/admin/pages/brands/form.blade.php]: it should be single `<form action` line

**Review by rajan-parmar — CHANGES_REQUESTED** [review]

**Review by rajan-parmar — APPROVED** [review]

File: app/Http/Requests/BrandRequest.php [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+{
+    public function authorize(): bool
+    {
+        return true;
+    }
+    public function rules(): array
+    {
+        return [
+                  'name' => ['required','string','max:255',Rule::unique('brands')->ignore($this->brand, 'name')],
+               ];
```
Comment (gauravmak) [inline app/Http/Requests/BrandRequest.php]: @jayesh-parmar Please take care with spacing everywhere

**Review by gauravmak — CHANGES_REQUESTED** [review]

**Pull request merged**